scraper/scripts/bootstrap.sh

```diff
@@ -34,6 +34,7 @@ pushd data
 
 # Compress the $BANK.json files
 tar --gzip --create --file by-bank.tar.gz by-bank
+tar cvf by-bank.tar by-bank
```

Comment:
why this?

Reply:
actually, in php tests, it's trying to decompress the .gz file and then using that tar file and processing the decompression is consuming a lot of memory and all, so instead created another file to not to decompress and reduce memory usage. getting [this error on the decompressing part.](https://app.wercker.com/Razorpay/ifsc/runs/php-composer-test/5f06075bfb1f8e000838fda0?step=5f060788aa6b94000801e60b). and am not copying that file to artifacts.